README.md

```diff
@@ -11,7 +11,7 @@
 Install:
 
 ```bash
-yarn add theme-colors
```

Comment:
I think we could update using the installation section in the template part. https://github.com/unjs/template?tab=readme-ov-file#usage